network-api/networkapi/mozfest/blocks/carousel_text_block.py

```diff
@@ -6,11 +6,7 @@
 class CarouselTextBlock(blocks.StructBlock):
     heading = blocks.CharBlock(help_text="Heading for the block.", required=False)
     text = blocks.RichTextBlock(features=["bold", "italic", "link"])
-    # Use specific link fields for the CTA on the block as opposed to the
-    # common.link_blocks.LabelledExternalLinkBlock so it can be marked as
-    # required=False.
```

Comment:
Noticed this comment. Since the LinkBlock is not required, and only the link block fields are required, this case appears to be satisfied 👍